tasks/npm.yml

```diff
@@ -1,48 +1,14 @@
 ---
-
-- name: Ensure npm has permissions to write
-  file:
-    state: directory
-    path: "{{ django_stack_npm_dir }}"
-    recurse: yes
-    owner: "{{ django_stack_gcorn_user }}"
-    group: "{{ django_stack_gcorn_group }}"
-  changed_when: false
+- name: Establish some npm specific vars for current play
+  set_fact:
+    npm_touch: "touch /status/npm.done"
```

Comment:
This tempfile logic is difficult to understand. Some comments would be warranted. Near as I can figure, this task is referencing a static location incorrectly, when it should be using `"{{ tmp_dir }}/npm.done" instead. More explicit var names would help here, e.g. `django_stack_local_tmp_dir`.